packages/xo-web/src/common/intl/messages.js

```diff
@@ -906,6 +906,7 @@ const messages = {
   hostLicenseSocket: 'Socket',
   hostLicenseExpiry: 'Expiry',
   hostRemoteSyslog: 'Remote syslog',
+  hostIommu: 'Iommu',
```

Comment:
Iommu is an acronym for "Input–output memory management unit". As is, it should be in caps, `IOMMU`.

For people curious about it, you can read https://en.wikipedia.org/wiki/Input%E2%80%93output_memory_management_unit